src/DIYManagementAPI/Controllers/DIYController.cs

```diff
@@ -46,6 +46,18 @@ public async Task<ActionResult<IEnumerable<DIYEveningModel>>> GetDIYEvening()
             return Ok(result);
         }
 
+        [HttpPut("{id}")]
```

Comment:
Ik zou graag voorstellen om hier een prefix met 'cancel' van te maken. dit omdat er in de toekomst misschien anderen put api's komen. dan ga je dus meerdere put's krijgen en geen duidelijke aanduiding van welke put. dit werkt;;

```suggestion
        [HttpPut("cancel/{id}")]
```

dus dan wordt de api; 

`http://localhost:5300/api/diy/cancel/2004`